src/instance.ts

```diff
@@ -192,6 +238,249 @@ class Instance extends common.GrpcServiceObject {
     this.databases_ = new Map<string, Database>();
   }
 
+  /**
+   * Get a reference to a Backup object.
+   *
+   * @throws {Error} If any parameter is not provided.
+   *
+   * @param {string} backupId The name of the backup.
+   * @param {string} databasePath the path of the backup.  Only needs to be specified for new backups.
+   * @param {PreciseDate} expireTime expiry time of the backup.  Only needs to be specified for new backups.
+   * @return {Backup} A Backup object.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const backupExpiryDate = new PreciseDate(Date.now() + 1000 * 60 * 60 * 24)
+   * const backup = instance.backup('my-backup', 'projects/my-project/instances/my-instance/databases/my-database', backupExpiryDate);
+   */
+  backup(
+    backupId: string,
+    databasePath: string,
```

Comment:
Can a backup be referenced without a database? If not, I would think we should move this accessor method to the Database class:

```ts
const spanner = new Spanner();
const instance = spanner.instance('my-instance');
const database = instance.database('my-database');
const backup = database.backup('my-backup');
```

Reply:
Yeah a backup can be referenced without a database. You only need to know the database if you want to create a backup. After that, the backup's association is only with the instance that it lives in. This is why we put this in the instance. It is implemented the same way in other languages as well.